services/keyword_service/keyword_test.go

```diff
@@ -87,6 +87,47 @@ func (s *KeywordServiceDbTestSuite) TestGetAllWithInvalidUser() {
 	assert.Equal(s.T(), nil, err)
 }
 
+func (s *KeywordServiceDbTestSuite) TestGetKeywordResultWithValidKeywordAndUser() {
+	keyword := models.Keyword{UserID: s.userID, Keyword: faker.Name()}
+	db.GetDB().Create(&keyword)
+
+	result, err := s.keywordService.GetKeywordResult(keyword.ID)
+
+	assert.Equal(s.T(), keyword.Keyword, result.Keyword)
+	assert.Equal(s.T(), nil, err)
+}
+
+func (s *KeywordServiceDbTestSuite) TestGetKeywordResultWithValidKeywordButInvalidUser() {
+	keyword := models.Keyword{UserID: s.userID, Keyword: faker.Name()}
+	db.GetDB().Create(&keyword)
+
+	keywordService := KeywordService{CurrentUserID: 999999}
+	result, err := keywordService.GetKeywordResult(keyword.ID)
+
+	assert.Equal(s.T(), models.Keyword{}, result)
+	assert.Equal(s.T(), "something went wrong, please try again", err.Error())
```

Comment:
IMO, it is not `Unauthorize` but it should be something like `keyword not found` because this is the function that query from database and it is not related to authentication

so I fixed the error message from `something went wrong, please try again` to `keyword not found` in 406cbbdb8458be74464f4342095d1f2458956f9f